root/commons/src/main/java/at/fhv/itb5c/commons/dto/ITournament.java

```diff
@@ -0,0 +1,39 @@
+package at.fhv.itb5c.commons.dto;
+
+import java.time.LocalDate;
+import java.util.Set;
+
+public interface ITournament {
+	void setId(Long id) throws Exception;
+
+	Long getId() throws Exception;
+
+	void setVersion(Long version) throws Exception;
+
+	Long getVersion() throws Exception;
+
+	String getName() throws Exception;
+
+	void setName(String name) throws Exception;
+
+	double getFee() throws Exception;
+
+	void setFee(double fee) throws Exception;
+
+	ITeam getHomeTeam() throws Exception;
+
+	void setHomeTeam(ITeam homeTeam) throws Exception;
+
+	Set<? extends ITeam> getGuestTeams() throws Exception;
```

Comment:
for the guest teams, we only have to store the names (--> `String`), not entire `ITeam`s